xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/commentsinline.vm

```diff
@@ -184,7 +184,7 @@ $xwiki.ssfx.use('uicomponents/viewers/comments.css', true)
       <a class="permalink btn btn-default btn-xs" data-toggle="modal" data-target="#permalinkModal" rel="nofollow"
         href="$doc.getURL('view', 'viewer=comments')#xwikicomment_${comment.number}"
         title="$services.localization.render('core.viewers.comments.permalink')">$services.icon.renderHTML('link')</a>
-      #if ($hasAdmin || ($hasEdit && $isUserComment))
+      #if ($hasAdmin || $isUserComment)
```

Comment:
What about introducing a new variable to use in both if for edit and delete? That way we ensure that we'll keep them consistent.

Reply:
Done in https://github.com/xwiki/xwiki-platform/pull/2836/commits/9f841ae94d968251156fe49ca69e011e87cf20cb :+1: